rick-and-morty/src/components/Characters.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Link } from 'react-router-dom'
+import './Characters.css'
+
+class Characters extends React.Component {
```

Comment:
I like how you used router + displayed the character component next to the list. Nice UI design 👍